cocoa/src/toga_cocoa/libs/foundation.py

```diff
@@ -51,3 +51,7 @@
 ######################################################################
 # NSValue.h
 NSNumber = ObjCClass("NSNumber")
+
+######################################################################
+# NSAutoreleasePool.h
+NSAutoreleasePool = ObjCClass("NSAutoreleasePool")
```

Comment:
Is this needed? See the related comment around the use of the autorelease pool.

Reply:
Removed it.